Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-09-13
Initial Package Version: 18
Origin: Rawhide
Description: Better Installation Routine

diff -Naur procinfo-18.orig/Makefile procinfo-18/Makefile
--- procinfo-18.orig/Makefile	2001-02-24 23:29:53.000000000 +0000
+++ procinfo-18/Makefile	2003-09-13 07:31:24.000000000 +0000
@@ -1,6 +1,7 @@
 ### Makefile for procinfo.
 
 prefix=/usr
+mandir=$(prefix)/man
 
 CC = gcc -Wall -Wstrict-prototypes
 
@@ -33,13 +34,13 @@
 
 install: procinfo procinfo.8 lsdev.pl socklist.pl lsdev.8
 	-mkdir -p $(prefix)/bin
-	install procinfo $(prefix)/bin/procinfo
-	install lsdev.pl $(prefix)/bin/lsdev
-	install socklist.pl $(prefix)/bin/socklist
-	-mkdir -p $(prefix)/man/man8
-	install -m 644  procinfo.8 $(prefix)/man/man8/procinfo.8
-	install -m 644  lsdev.8 $(prefix)/man/man8/lsdev.8
-	install -m 644  socklist.8 $(prefix)/man/man8/socklist.8
+	install -m 755 procinfo $(prefix)/bin/procinfo
+	install -m 755 lsdev.pl $(prefix)/bin/lsdev
+	install -m 755 socklist.pl $(prefix)/bin/socklist
+	-mkdir -p $(mandir)
+	install -m 644  procinfo.8 $(mandir)/man8/procinfo.8
+	install -m 644  lsdev.8 $(mandir)/man8/lsdev.8
+	install -m 644  socklist.8 $(mandir)/man8/socklist.8
 
 clean:
 	rm -f procinfo procinfo.0 *.o *~ out
